src/views/sidebar.js

```diff
@@ -70,6 +83,42 @@ class Sidebar extends Component {
             {pubKey}
           </Text>
         </TouchableOpacity>
+        {syncedToChain ? null : (
+          <div
+            style={{
+              padding: 10,
+              paddingBottom: 12,
+              textAlign: 'center',
+              backgroundColor: colors.blue,
+            }}
+            className="syncing"
+          >
+            <style
+              dangerouslySetInnerHTML={{
```

Comment:
Do we really need/want to `dangerouslySetInnerHTML` ?

Reply:
This was copied straight from v1. On second thought, is not really necessary. Will fix.